src/settings_translation_file.cpp

```diff
@@ -762,6 +762,46 @@ fake_function() {
 	gettext("Slope and fill work together to modify the heights.");
 	gettext("Dungeon noise");
 	gettext("3D noise that determines number of dungeons per mapchunk.");
+	gettext("Mapgen Trailgen");
+	gettext("Mapgen Trailgen specific flags");
+	gettext("Map generation attributes specific to Mapgen trailgen.");
+	gettext("Large cave depth");
+	gettext("Y of upper limit of large caves.");
+	gettext("Small cave minimum number");
+	gettext("Minimum limit of random number of small caves per mapchunk.");
+	gettext("Small cave maximum number");
+	gettext("Maximum limit of random number of small caves per mapchunk.");
+	gettext("Large cave minimum number");
+	gettext("Minimum limit of random number of large caves per mapchunk.");
+	gettext("Large cave maximum number");
+	gettext("Maximum limit of random number of large caves per mapchunk.");
+	gettext("Large cave proportion flooded");
+	gettext("Proportion of large caves that contain liquid.");
+	gettext("Cave width");
+	gettext("Controls width of tunnels, a smaller value creates wider tunnels.\nValue >= 10.0 completely disables generation of tunnels and avoids the\nintensive noise calculations.");
+	gettext("Cavern limit");
+	gettext("Y-level of cavern upper limit.");
+	gettext("Cavern taper");
+	gettext("Y-distance over which caverns expand to full size.");
+	gettext("Cavern threshold");
+	gettext("Defines full size of caverns, smaller values create larger caverns.");
+	gettext("Adjust Map Generation Height");
+	gettext("Adjust map generation height.\nWhen joining trailgen and v6 worlds together input a 2 for a smoother transition between the two mapgen types.");
+	gettext("Dungeon minimum Y");
+	gettext("Lower Y limit of dungeons.");
+	gettext("Dungeon maximum Y");
+	gettext("Upper Y limit of dungeons.");
+	gettext("Noises");
+	gettext("Terrain base noise");
+	gettext("Y-level of lower terrain and seabed.");
+	gettext("Terrain higher noise");
+	gettext("Y-level of higher terrain that creates cliffs.");
+	gettext("Steepness noise");
+	gettext("Varies steepness of cliffs.");
+	gettext("Height select noise");
+	gettext("Defines distribution of higher terrain.");
+	gettext("Filler Depth");
+	gettext("Variation of biome filler depth.");
```

Comment:
this file is regenerated once in a while, no need to apply any changes now.

Reply:
Ah okay thank you.